### ISSUE TITLE
RE3: Option to prevent run animation triggering while aiming weapon

### ISSUE THREAD

**Issue by DaveACG** [praydog/REFramework#901]
For controllers, RE3 has the run hold/toggle mapped to the left thumbstick depress. 

While this is not an uncommon mapping in many games, what is unusual in RE3's case is that depressing the thumbstick actually causes the character to go into a run animation - regardless if you're already engaging another action (or two!) that requires the triggers to be held (such as weapon zoom/shooting). It acts less like a walk/run toggle and more like a 'character run mode engage' that overrides whatever mode the character is already in the second it's pressed.

This can be extremely frustrating during gameplay. For example: You can be inching along a corridor with your weapon zoomed, waiting to shoot a licker - but in the middle of shooting, if you jerk your thumbstick too hard in one direction and accidentally depress it (which especially for Playstation controllers is very easy to do), your character will suddenly stop shooting and start running - despite both the left and right triggers remaining depressed. The only way to get out of this state is to let go of the thumbstick and the triggers, and then try again and hope you don't depress the thumbstick again. This has led to a _lot_ of deaths for me.

Basically if possible, some way to set a config option whereby the game will ignore run requests while either trigger is already depressed.
